tools/bgpreader.c

```diff
@@ -667,7 +668,17 @@ int main(int argc, char *argv[])
     }
 
     if (record_bgpdump_output_on) {
-      bgpstream_record_print_mrt_data(bs_record);
+      // print record following bgpdump format
```

Comment:
i think this is not going to work if the user runs `bgpreader -e -m ...`

Reply:
attempt to resolve this in https://github.com/CAIDA/libbgpstream/commit/5980312af98bb4364ef2aa87d2287bcc0a3fb13d